doc-content/enterprise-only/employee-rostering/er-deploy-script-proc.adoc

```diff
@@ -25,14 +25,16 @@ You can deploy the {PLANNER} Employee Rostering starter application to {OPENSHIF
 If the current OpenShift user name contains characters that are not letters or numbers, the deployment might fail. You can resolve this issue by providing an additional parameter: `./provision.sh setup employee-rostering --binary --project-suffix optaweb`.  This example uses "optaweb", but you can use any value for the suffix.
 ====
 +
-Compilation and packaging might take several minutes to complete. They  continually show progress on the command line output.
+Compilation and packaging might take up to 10 minutes to complete. They  continually show progress on the command line output.
```

Comment:
The IBM style guide bans "may". Page 344:

may v
Use only to indicate permission. Otherwise, use “can” or “might.”
“May” is overused in technical information and should in most instances be changed to “can” or “might.”

So my choice here is "can" or "might". As this time strongly depends on the computing power on that particular host, I think "might" is more appropriate.

Changed "they" to "these processes".